sdk/client.py

```diff
@@ -114,22 +117,25 @@ def _send_batch(
         jobs_data = batch_data.pop("jobs", [])
         return batch_data, jobs_data
 
-    def _complete_batch(self, batch_id: int):
-        response = self._request(
+    def _complete_batch(self, batch_id: int) -> Dict[str, Any]:
+        response: Dict[str, Any] = self._request(
             "PUT", f"{self.endpoints.core}/api/v1/batches/{batch_id}/complete"
         )["data"]
         return response
 
-    def _send_job(self, job_data: Dict):
-        return self._request("POST", f"{self.endpoints.core}/api/v1/jobs", job_data)[
-            "data"
-        ]
+    def _send_job(self, job_data: Dict[str, Any]) -> Dict[str, Any]:
+        response: Dict[str, Any] = self._request(
+            "POST", f"{self.endpoints.core}/api/v1/jobs", job_data
+        )["data"]
+        return response
 
-    def _get_batch(self, id: int, fetch_results: bool = False) -> Tuple[Dict, Dict]:
-        batch_data = self._request("GET", f"{self.endpoints.core}/api/v1/batches/{id}")[
-            "data"
-        ]
-        jobs_data = batch_data.pop("jobs", {})
+    def _get_batch(
+        self, id: int, fetch_results: bool = False
+    ) -> Tuple[Dict[str, Any], List[Dict[str, Any]]]:
+        batch_data: Dict[str, Any] = self._request(
+            "GET", f"{self.endpoints.core}/api/v1/batches/{id}"
+        )["data"]
+        jobs_data = batch_data.pop("jobs", [])
```

Comment:
Looks like before _get_batch and send batch had different default types.
Now they both have `List`, as they should.
Cf line 117